src/librustdoc/html/render.rs

```diff
@@ -4063,11 +4063,25 @@ fn get_methods(
         .filter_map(|item| match item.name {
             Some(ref name) if !name.is_empty() && item.is_method() => {
                 if !for_deref || should_render_item(item, deref_mut) {
-                    Some(format!(
-                        "<a href=\"#{}\">{}</a>",
-                        get_next_url(used_links, format!("method.{}", name)),
-                        name
-                    ))
+                    let mut emojis = String::new();
+                    if let clean::FunctionItem(ref func) | clean::ForeignFunctionItem(ref func) = item.inner {
+                        if func.header.unsafety == hir::Unsafety::Unsafe {
+                            emojis += "⚠";
+                        }
+                    }
+                    if let Some(stab) = item.stability.as_ref().filter(|stab| stab.level == stability::Unstable) {
```

Comment:
```suggestion
                    if let Some(stab) = item.stability.as_deref().filter(|stab| stab.level == stability::Unstable) {
```

Reply:
@lzutao Can I know how is it different?

Reply:
I think it make the intent more clear as we are using its dereference.

Reply:
Looks like  we need to change it back. Sorry I just don't know which type of `item.stability`.

Reply:
Okay, no worries.